metadata_manager/core.py

```diff
@@ -350,14 +352,33 @@ def reload_remotes_json(self) -> None:
         )
         with open(self.__remotes_json_path, 'r') as f, \
              open(remotes_json_schema_path, 'r') as s:
```

Comment:
Isn't there an indenting problem here?

Reply:
I don't think so. I might look due to the backslash. It works fine, I have tested it.